abci/types/types_test.go

```diff
@@ -49,21 +49,13 @@ func TestHashDeterministicFieldsOnly(t *testing.T) {
 	tr1 := abci.ExecTxResult{
 		Code:      1,
 		Data:      []byte("transaction"),
-		Log:       "nondeterministic data: abc",
-		Info:      "nondeterministic data: abc",
 		GasWanted: 1000,
-		GasUsed:   1000,
-		Events:    []abci.Event{},
 		Codespace: "nondeterministic.data.abc",
 	}
```

Comment:
Isn't `ExecTxResult` part of `FinalizeBlock` and not `CheckTx`

Reply:
Reverted